### ISSUE TITLE
Pebble 4.6.2 import error on Windows

### ISSUE THREAD

**Issue by saadin** [noxdafox/pebble#84]
On windows, importing pebble 4.6.2 gives the following error. (version 4.6.1 works fine on windows)
```
>>> import pebble
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python39\lib\site-packages\pebble\__init__.py", line 21, in <module>
    from pebble.pool import ThreadPool, ProcessPool, MapFuture, ProcessMapFuture
  File "C:\Python39\lib\site-packages\pebble\pool\__init__.py", line 8, in <module>
    from pebble.pool.process import ProcessPool
  File "C:\Python39\lib\site-packages\pebble\pool\process.py", line 33, in <module>
    from pebble.pool.channel import ChannelError, channels
  File "C:\Python39\lib\site-packages\pebble\pool\channel.py", line 194, in <module>
    READ_ONLY_EVENTMASK = select.POLLIN | select.POLLPRI | select.POLLHUP | select.POLLERR
AttributeError: module 'select' has no attribute 'POLLIN'
```

**Comment by noxdafox** [comment]
Thanks for reporting this!

Indeed we did not verify the change against Windows. 
There used to be Travis job which would have let us see this but Travis CI eventually shut down. :disappointed: 
I am moving to GH actions and in the process of figuring out Windows and MAC test automation.

You can stick to `4.6.1` in the meanwhile as the changes in `4.6.2` mostly affect certain versions of MAC OS. I will fix this ASAP.

**Comment by noxdafox** [comment]
Issue resolved in `4.6.3`.

MAC OS and Windows builds are back online as well.
